src/pymor/algorithms/eigs.py

```diff
@@ -104,6 +104,9 @@ def eigs(A, E=None, k=3, sigma=None, which='LM', b=None, l=None, maxiter=1000, t
     if l is None:
         l = min(n - 1, max(2 * k + 1, l_min))
 
+    if k == n:
+        return A.apply(E.apply_inverse(E.source.ones(1))).to_numpy()
+
```

Comment:
This does not make any sense to me. Assuming `E` is the identity, this would just return `A` as a matrix. This clearly isn't an eigendecomposition of `A`. Also, `eigs` has two return values.

Reply:
I have no idea how that has gotten there and what was the thought process behind it. I have removed it.